setup/install/providers/ecs.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: single
+title:  "ECS (Amazon Web Services)"
+sidebar:
+  nav: setup
+redirect_from: /setup/providers/ecs/
+---
+
+{% include toc %}
+
+In the ECS cloud provider, an [__Account__](/concepts/providers/#accounts)
+maps to a Spinnaker AWS account, which itself is able to authenticate against a given [AWS
+account](https://aws.amazon.com/account/).
+
+## Prerequisites
+
+### ECS Cluster
+You need to [create an ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create_cluster.html) and have enough capacity in it to deploy your containers.
+Please note that ECS clusters are not the same concept as [Spinnaker clusters](/concepts/clusters/).  
+
+### Networking 
+As Elastic Network Interfaces (ENIs) are not yet supported in Spinnaker, you do not need to setup any further networking.  The cluster's networking configuration will be passed from your cluster instances to your containers.   
+
+### Spinnaker Clouddriver role 
+
+The role that Clouddriver assumes for your ECS account needs to have the following trust relationships:
+```yaml
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+                "Service": [
+                  "ecs.amazonaws.com",
+                  "application-autoscaling.amazonaws.com"
+                ],
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
```

Comment:
Can you say more about this. Like, which yaml is this and where does it live?

Reply:
Adding some context: `The role that Clouddriver assumes for your ECS account needs to have the following trust relationships for your Spinnaker IAM assumed role`

Reply:
@BrunoCarrier I'd still like to see the name of the yml file and where it lives. I think it's on the ECS side, right? Not Spinnaker. But it's not clear to me.

Reply:
Just below I link to the AWS setup which includes setting the trust relationships for the role.  The concept of a `trust relationship` is a quite common one, and is purely on the cloud provider side - it is not created /managed on the spinnaker side of things


I took the paragraph below the JSON example and moved it above the JSON example in a bid to make this more clear.  

If this is not clear, could you suggest a way to make it more clear?  I am not sure really how to go at it if this doesn't help it :)